docs/quick_start/AA_hexane_ethanol_example.rst

```diff
@@ -0,0 +1,40 @@
+Example of an All-Atom (AA) Hexane and Ethanol System
+========================
+
+**Note**: `foyer <https://foyer.mosdef.org/en/stable/>`_ is used in conjunction with **mBuild** in the following example to demonstrate how all the `MoSDeF <https://mosdef.org>`_ software integrates to generate a simulation box.
+
+Import the required mbuild package.
+
+.. code:: ipython3
+
+    import mbuild as mb
+
+
+Construct an all-atom (AA) hexane and ethanol using the OPLS-AA force field (FF),
+which is comes as a standard `foyer <https://foyer.mosdef.org/en/stable/>`_ force field (FF).
+The hexane and ethanol residues will be named HEX and ETO, respectively.  
+Lastly, the hexane and ethanol molecule's configuration will be energy minimized. 
+
+**Note**: The energy minimize step requires the `foyer <https://foyer.mosdef.org/en/stable/>`_ package. 
+
+.. code:: ipython3
+
+    hexane = mb.load('CCCCCC', smiles=True, name='HEX')
+    hexane.name = 'HEX'
+    hexane.energy_minimize(forcefield='oplsaa' , steps=10**4)
+
+
+    ethanol = mb.load('CCO', smiles=True)
```

Comment:
mention that you are loading a smiles string and give a little info about what that is (maybe link to the smiles website)

Reply:
added link